classes/class3/homework/submit_casts/routes/index.js

```diff
@@ -0,0 +1,75 @@
+var mongoose = require('mongoose');
+
+var mongoURI = process.env.MONGOURI || "mongodb://localhost/test";
+mongoose.connect(mongoURI);
+
+
+var castSchema = mongoose.Schema({
```

Comment:
I know it's just a couple lines now, but in the future, break your schemas out in to a `models` directory and then require them here.